From 8e20e2647e5592270db051c206f9ef788513f559 Mon Sep 17 00:00:00 2001
From: Xianzhong <b07117@freescale.com>
Date: Thu, 3 May 2012 01:34:05 +0800
Subject: [PATCH 270/635] ENGR00174836 revert YUV mismatch usage workaround

Revert the workaround introduced by commit:
75af119cc59a90d5beefb5b4023980e272ae9090

Signed-off-by: Xianzhong <b07117@freescale.com>
---
 mx6/libgralloc_wrapper/gralloc.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/mx6/libgralloc_wrapper/gralloc.cpp b/mx6/libgralloc_wrapper/gralloc.cpp
index 6454935..56c4f44 100755
--- a/mx6/libgralloc_wrapper/gralloc.cpp
+++ b/mx6/libgralloc_wrapper/gralloc.cpp
@@ -114,10 +114,6 @@ static int gralloc_alloc(alloc_device_t* dev,
         return -EINVAL;
     }
 
-    //Correct android YUV 420 format mess usage by players
-    if(format == HAL_PIXEL_FORMAT_YCbCr_420_I)format = HAL_PIXEL_FORMAT_YCbCr_420_P;
-    else if(format == HAL_PIXEL_FORMAT_YCbCr_420_P)format = HAL_PIXEL_FORMAT_YCbCr_420_I;
-
     return m->gpu_device->alloc(dev, w, h, format, usage, pHandle, pStride);
 }
 
-- 
1.8.0

